views/layout.erb

```diff
@@ -0,0 +1,22 @@
+<html>
+
+  <body>
+  <header>
+  
+  <% if session[:user_id] != nil %>
+
+  <a href="/spaces"> Spaces </a> | Requests | <a href="/logout"> Sign out </a>
```

Comment:
the mock up says the non-user view should be About and Log in and the user view should be Spaces, Requests and Sign out. Maybe add those in too.